pennylane/data/base/hdf5.py

```diff
@@ -96,15 +96,14 @@ def copy_all(
         dest.attrs.update(source.attrs)
 
 
-def open_hdf5_s3(s3_url: str, cache_dir: Optional[Path] = None) -> HDF5Group:
+def open_hdf5_s3(s3_url: str) -> HDF5Group:
     """Uses ``fsspec`` module to open the HDF5 file at ``s3_url``.
 
     This requires both ``fsspec`` and ``aiohttp`` to be installed.
     """
 
-    if cache_dir is not None:
-        fs = fsspec.open(f"blockcache::{s3_url}", blockcache={"cache_storage": str(cache_dir)})
-    else:
-        fs = fsspec.open(s3_url)
+    # Tells fsspec to fetch data in 8MB chunks for faster loading
```

Comment:
Is this the maximum `fsspec` allows? Should we allow users to change this if they want something faster? 🤔 
The rest looks good to me.

Reply:
It can be bigger but you get diminishing returns - a bigger chunk size will also make small reads (metadata, checking attributes etc) slower. 8MB seems to be a good compromise. I've added a `block_size` parameter to `load()`